monai/transforms/io/array.py

```diff
@@ -335,6 +338,13 @@ def __init__(
         )
 
         self.output_ext = output_ext.lower() or output_format.lower()
+        if isinstance(writer, str):
+            writer_, has_built_in = optional_import("monai.data", name=f"{writer}")  # search built-in
+            if not has_built_in:
+                writer_ = locate(f"{writer}")  # search dotted path
```

Comment:
Could you please help add a simple unit test to cover this feature?
I think it would be pretty useful for BYOC writers.

Thanks in advance.

Reply:
yes it's tested with the `"monai.data.NibabelWriter"` test case, this string works fine because of `locate(writer)`